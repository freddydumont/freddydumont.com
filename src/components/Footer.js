import React from 'react';
import FooterWave from './FooterWave';
import { Box, Container, Text } from '@theme-ui/components';

const Footer = () => {
  return (
    <>
      <FooterWave />
      <Box as="footer" bg="gray.8" pb={[16, 32]}>
        <Container>
          <Text
            as="h2"
            variant="heading"
            pt={2}
            mb={4}
            sx={{
              fontSize: ['2xl', '4xl'],
            }}
          >
            Elsewhere on the internet
          </Text>
        </Container>
      </Box>
    </>
  );
};

export default Footer;
