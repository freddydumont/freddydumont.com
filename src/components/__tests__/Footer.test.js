import React from 'react';
import { render } from '../../../test/test-utils';
import Footer from '../Footer';

describe('Footer', () => {
  it('should match snapshot', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
