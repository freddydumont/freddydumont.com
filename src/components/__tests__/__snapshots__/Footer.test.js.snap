// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
@media screen and (min-width:768px) {
  .emotion-1 {
    margin-top: -2rem;
  }
}

@media screen and (min-width:1024px) {
  .emotion-1 {
    margin-top: -4rem;
  }
}

.emotion-0 {
  fill: var(--theme-ui-colors-gray-8,#3B3B3B);
}

.emotion-17 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  padding-bottom: 1rem;
  background-color: var(--theme-ui-colors-gray-8,#3B3B3B);
}

.emotion-16 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  width: 100%;
  max-width: 20rem;
  margin-left: auto;
  margin-right: auto;
  padding-right: 1.5rem;
  padding-left: 1.5rem;
  max-width: 32rem;
}

@media screen and (min-width:640px) {
  .emotion-16 {
    max-width: 32rem;
  }
}

@media screen and (min-width:768px) {
  .emotion-16 {
    max-width: 42rem;
  }
}

@media screen and (min-width:1024px) {
  .emotion-16 {
    max-width: 56rem;
  }
}

.emotion-15 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  text-align: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-size: 2.25rem;
  color: var(--theme-ui-colors-gray-1,#E1E1E1);
  font-weight: 600;
  padding-top: 1rem;
  margin-bottom: 1.25rem;
  font-size: 1.5rem;
}

@media screen and (min-width:640px) {
  .emotion-2 {
    font-size: 2.25rem;
  }
}

@media screen and (min-width:768px) {
  .emotion-2 {
    font-size: 3rem;
  }
}

@media screen and (min-width:640px) {
  .emotion-2 {
    font-size: 2.25rem;
  }
}

@media screen and (min-width:768px) {
  .emotion-2 {
    font-size: 2.25rem;
  }
}

@media screen and (min-width:1024px) {
  .emotion-2 {
    font-size: 2.25rem;
  }
}

.emotion-12 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin-bottom: 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin-left: 0.75rem;
  margin-right: 0.75rem;
}

.emotion-4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0.25rem;
  width: 8rem;
  height: 8rem;
  color: inherit;
  background-color: var(--theme-ui-colors-transparent,transparent);
  border: none;
  border-radius: 4px;
  color: var(--theme-ui-colors-gray-1,#E1E1E1);
  width: 2rem;
  height: 2rem;
  padding: 0;
}

@media screen and (min-width:640px) {
  .emotion-4 {
    width: 2.5rem;
    height: 2.5rem;
  }
}

.emotion-4:hover {
  color: var(--theme-ui-colors-primaryHover,#2DCCA7);
}

.emotion-3 {
  width: inherit;
  height: inherit;
  -webkit-transition: all 100ms ease-in-out;
  transition: all 100ms ease-in-out;
}

.emotion-13 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 1rem;
  font-family: "Overpass Mono",Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
  color: var(--theme-ui-colors-gray-1,#E1E1E1);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-13 li {
  margin-left: 0.75rem;
  margin-right: 0.75rem;
}

.emotion-13 li:hover {
  color: var(--theme-ui-colors-primaryHover,#2DCCA7);
}

.emotion-14 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  color: var(--theme-ui-colors-gray-1,#E1E1E1);
  font-family: "Overpass Mono",Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
}

<div>
  <svg
    class="emotion-1"
    preserveAspectRatio="none"
    viewBox="0 0 1440 320"
  >
    <path
      class="emotion-0"
      d="M0,256L80,261.3C160,267,320,277,480,272C640,267,800,245,960,213.3C1120,181,1280,139,1360,117.3L1440,96L1440,320L1360,320C1280,320,1120,320,960,320C800,320,640,320,480,320C320,320,160,320,80,320L0,320Z"
    />
  </svg>
  <footer
    class="emotion-17"
  >
    <div
      class="emotion-16"
    >
      <div
        class="emotion-15"
      >
        <h2
          class="emotion-2"
        >
          Elsewhere on the internet
        </h2>
        <div
          class="emotion-12"
        >
          <a
            class="emotion-5"
            href="https://twitter.com/_freddydumont"
            rel="noreferrer noopener"
            target="_blank"
          >
            <button
              class="emotion-4"
            >
              <svg
                class="emotion-3"
                fill="currentColor"
                viewBox="0 0 20 20"
              >
                <title>
                  Twitter
                </title>
                <path
                  d="M6.29 18.25c7.55 0 11.67-6.25 11.67-11.67v-.53c.8-.59 1.49-1.3 2.04-2.13-.75.33-1.54.55-2.36.65a4.12 4.12 0 001.8-2.27c-.8.48-1.68.81-2.6 1a4.1 4.1 0 00-7 3.74 11.65 11.65 0 01-8.45-4.3 4.1 4.1 0 001.27 5.49C2.01 8.2 1.37 8.03.8 7.7v.05a4.1 4.1 0 003.3 4.03 4.1 4.1 0 01-1.86.07 4.1 4.1 0 003.83 2.85A8.23 8.23 0 010 16.4a11.62 11.62 0 006.29 1.84"
                />
              </svg>
            </button>
          </a>
          <a
            class="emotion-5"
            href="https://github.com/freddydumont"
            rel="noreferrer noopener"
            target="_blank"
          >
            <button
              class="emotion-4"
            >
              <svg
                class="emotion-3"
                fill="currentColor"
                viewBox="0 0 20 20"
              >
                <title>
                  GitHub
                </title>
                <path
                  d="M10 0a10 10 0 00-3.16 19.49c.5.1.68-.22.68-.48l-.01-1.7c-2.78.6-3.37-1.34-3.37-1.34-.46-1.16-1.11-1.47-1.11-1.47-.9-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.9 1.52 2.34 1.08 2.91.83.1-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.94 0-1.1.39-1.99 1.03-2.69a3.6 3.6 0 01.1-2.64s.84-.27 2.75 1.02a9.58 9.58 0 015 0c1.91-1.3 2.75-1.02 2.75-1.02.55 1.37.2 2.4.1 2.64.64.7 1.03 1.6 1.03 2.69 0 3.84-2.34 4.68-4.57 4.93.36.31.68.92.68 1.85l-.01 2.75c0 .26.18.58.69.48A10 10 0 0010 0"
                />
              </svg>
            </button>
          </a>
          <a
            class="emotion-5"
            href="https://www.linkedin.com/in/freddydumont/"
            rel="noreferrer noopener"
            target="_blank"
          >
            <button
              class="emotion-4"
            >
              <svg
                class="emotion-3"
                fill="currentColor"
                viewBox="0 0 382 382"
              >
                <title>
                  LinkedIn
                </title>
                <path
                  d="M347.445,0H34.555C15.471,0,0,15.471,0,34.555v312.889C0,366.529,15.471,382,34.555,382h312.889 C366.529,382,382,366.529,382,347.444V34.555C382,15.471,366.529,0,347.445,0z M118.207,329.844c0,5.554-4.502,10.056-10.056,10.056 H65.345c-5.554,0-10.056-4.502-10.056-10.056V150.403c0-5.554,4.502-10.056,10.056-10.056h42.806 c5.554,0,10.056,4.502,10.056,10.056V329.844z M86.748,123.432c-22.459,0-40.666-18.207-40.666-40.666S64.289,42.1,86.748,42.1 s40.666,18.207,40.666,40.666S109.208,123.432,86.748,123.432z M341.91,330.654c0,5.106-4.14,9.246-9.246,9.246H286.73 c-5.106,0-9.246-4.14-9.246-9.246v-84.168c0-12.556,3.683-55.021-32.813-55.021c-28.309,0-34.051,29.066-35.204,42.11v97.079 c0,5.106-4.139,9.246-9.246,9.246h-44.426c-5.106,0-9.246-4.14-9.246-9.246V149.593c0-5.106,4.14-9.246,9.246-9.246h44.426 c5.106,0,9.246,4.14,9.246,9.246v15.655c10.497-15.753,26.097-27.912,59.312-27.912c73.552,0,73.131,68.716,73.131,106.472 L341.91,330.654L341.91,330.654z"
                />
              </svg>
            </button>
          </a>
        </div>
        <ul
          class="emotion-13"
        >
          <li>
            <button>
              about
            </button>
          </li>
          <li>
            <a
              href="blog"
            >
              blog
            </a>
          </li>
          <li>
            <button>
              portfolio
            </button>
          </li>
          <li>
            <button>
              contact
            </button>
          </li>
          <li>
            <button>
              back to top
            </button>
          </li>
        </ul>
        <div
          class="emotion-14"
        >
          Copyright Â© 2020 freddydumont
        </div>
      </div>
    </div>
  </footer>
</div>
`;
